stock-app/src/test/java/edu/JasonFlinkstrom/advancedjava/stock/StockQuoteTest.java

```diff
@@ -0,0 +1,31 @@
+package edu.JasonFlinkstrom.advancedjava.stock;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+* Unit test for StockQuote class
+ */
+public class StockQuoteTest {
+
+    /**
+    * Positive test verifies given string is returned from getSymbol method
+     */
+    @Test
+    public void getSymbolpPosTest() {
+        String s = "APPL";
+        StockQuote stockQuote = new StockQuote(s);
+        assertTrue(stockQuote.getSymbol().equals(s));

```

Comment:
always use the form of assert that takes a diagnostic string that describes the test.